### PR TITLE
Partners Registry

### PR DESCRIPTION
**Pull request by ncomerci**
image

### CODE REVIEW

File: pages/components/PartnerCard/PartnerCard.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      <div
+        className={styles.Image}
+        style={{
+          background: `url(${DATA_URL}/assets/${partner.logo})`,
+        }}
+      >
+        <a href={WEBSITE} target="_blank" rel="noreferrer"></a>
+      </div>
+      <div className={styles.Container}>
+        <h3 style={{ margin: '0', marginBottom: '0.2rem' }}>
```
Comment (andyesp) [inline pages/components/PartnerCard/PartnerCard.tsx:39]: CSS please 🙏

File: pages/components/PartnerCard/PartnerCard.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  partner: VerifiedPartner
+}
+
+const DATA_URL = process.env.NEXT_PUBLIC_PARTNERS_DATA_URL
+
+function PartnerCard({ partner }: Props) {
+  const WEBSITE = partner.website || ''
+
+  return (
+    <div className={styles.PartnerCard}>
```
Comment (andyesp) [inline pages/components/PartnerCard/PartnerCard.tsx:29]: CSS module are scoped to the module right? So maybe this "global/main" classes we can call them Container? Or something like that.

Comment (andyesp) [reply pages/components/PartnerCard/PartnerCard.tsx:29]: Also we can write them in lowercase... so `styles.container`, `styles.image`. Somehow the first letter capitalized feels wrong in this context 😓

File: pages/interfaces/VerifiedPartner.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  opensea: null | string
+  sketchfab: null | string
+  parcel: null | string
+  marketplace: null | string
+  status: Status
+  user: string
+}
+
+export enum PaymentMethod {
+  CreditCard = "Credit Card",
```
Comment (andyesp) [inline pages/interfaces/VerifiedPartner.ts:34]: Are all these enum mappings just to show texts for the UI? It'd be great to add them to the locale file.

File: pages/components/PartnerCard/PartnerCard.tsx (hunk 1 above)
Comment (ncomerci) [reply pages/components/PartnerCard/PartnerCard.tsx:39]: Copy & paste 😅

File: pages/interfaces/VerifiedPartner.ts (hunk 3 above)
Comment (ncomerci) [reply pages/interfaces/VerifiedPartner.ts:34]: These Enums come from the API, so it seems to me that it does not make sense to add them to the locale 🤔

**Review by andyesp — APPROVED** [review]

**Pull request merged**